pagopa/pn-frontend
# PN-7003-fe-persona-fisica-creazione-delega-aggiunta-frase-sulla-validita-del-codice

## Short description
Changed italian sentences into deleghe of PG and PF.

## List of changes proposed in this pull request
- changed `show_code_subtitle` and `verificationCodeDescr`  with: 
`Condividi questo codice con la persona delegata: dovrà inserirlo all’accettazione della delega. Il codice è valido per 7 giorni dalla creazione della richiesta di delega.`

## How to test
Just `yarn start` pf/pg and see if the sentence of both attributes have been changed.
